src/service/components/index.js

```diff
@@ -18,17 +41,17 @@ const Default = new Map();
  * @param {string} name - The module name
  * @return {*} The default instance of a component
  */
-function acquire(name) {
+export function acquire(name) {
     let component;
 
     try {
         let info = Default.get(name);
 
         if (info === undefined) {
-            const module = imports.service.components[name];
+            const module = components[name];
 
             info = {
-                instance: new module.Component(),
+                instance: new module.default(),
```

Comment:
Now telephony's attempt to `acquire()` an instance of the `pulseaudio` component module (which exports its `Mixer`) as its `_mixer` object seems to be ending up with a `null`:

```js
const TelephonyPlugin = GObject.registerClass({
    GTypeName: 'GSConnectTelephonyPlugin',
}, class TelephonyPlugin extends Plugin {

    _init(device) {
        super._init(device, 'telephony');

        // Neither of these are crucial for the plugin to work
        this._mpris = Components.acquire('mpris');
        this._mixer = Components.acquire('pulseaudio');
    }
```

Or perhaps `acquire()` needs to be an `async` function?

Reply:
I think this is fixed by my comment [here](https://github.com/GSConnect/gnome-shell-extension-gsconnect/pull/1683#issuecomment-1963095964)